StripeIdentity/StripeIdentityTests/Snapshot/BiometricConsentViewControllerSnapshotTest.swift

```diff
@@ -17,7 +17,7 @@ final class BiometricConsentViewControllerSnapshotTest: FBSnapshotTestCase {
     override func setUp() {
         super.setUp()
 
-        //        recordMode = true
+//                recordMode = true
```

Comment:
Should we just delete this line?

Reply:
we use this pattern for other snapshot test as well - https://confluence.corp.stripe.com/display/MOBILE/Snapshot+Tests

> My only comment is that I am not a fan of the phone number dial pad being used to enter a date of birth.

hmm we're using `asciiCapableNumberPad`, it seems to be the value used for other similar fields(e.g the expiration date of a credit card)